e2e/test/scenarios/dashboard-filters/dashboard-filters-clear-and-restore.cy.spec.ts

```diff
@@ -1,27 +1,20 @@
 import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
 import {
+  checkFilterListSourceHasValue,
+  createQuestionAndDashboard,
   editDashboard,
+  modal,
```

Comment:
WDYT about auto formatting in ascending order like we do for the regular js/ts imports?
Maybe a good project for devex?

Reply:
Yes, please :pray: cc: @uladzimirdev

Reply:
done https://github.com/metabase/metabase/pull/47009